src/pkg/caendr/caendr/services/sql/etl/strain_annotated_variants.py

```diff
@@ -47,41 +51,70 @@ def parse_strain_variant_annotation_data(species, sva_fname: str):
       if idx % 1000000 == 0:
         logger.debug(f"Processed {idx} lines")
 
+      # Map row to dict, using file headers as keys
+      row = {
+        header: row[column_header_map[header]] for header in column_header_map
+      }
+
       target_consequence = None
-      consequence = row[4] if row[4] else None
+      consequence = row.get('CONSEQUENCE')
       alt_target = re.match('^@[0-9]*$', consequence)
       if alt_target:
         target_consequence = int(consequence[1:])
         consequence = None
 
       # Yield the row as a dict
       yield {
-        'id': idx,
-        'chrom': row[0],
-        'pos': int(row[1]) if row[1] else None,
-        'ref_seq': row[2] if row[2] else None,
-        'alt_seq': row[3] if row[3] else None,
-        'consequence': consequence,
+        'id':                 start_idx + idx,
+        'species_name':       species.name,
+        'chrom':              row['CHROM'],
```

Comment:
potential for normalization of access across three variants (`row`: [], .get(), get_row()).